content/docs/command-reference/remote/modify.md

```diff
@@ -758,6 +758,43 @@ more information.
 
 </details>
 
+<details>
+
+### Click for OSF
+
+- `url` - remote location:
+
+  ```dvc
+  $ dvc remote modify myremote url \
+        osf://odf.io/path/to/dir
+
+- `project` - [OSF](https://osf.io/) project name. You can see it in the `url` line
+  of your project page.
+
+  ```dvc
+  $ dvc remote modify myremote project <project_name>
+  
+- `osf_username` - username (e-mail) that you specify during registration.
+  ```dvc
+  $ dvc remote modify myremote osf_username \
+        example@mail.com
+  ```
```

Comment:
I fixed a bunch of but some are pending like this:
```suggestion
- `osf_username` - OSF username (email address):

  ```dvc
  $ dvc remote modify myremote osf_username \
        example@mail.com
  ```

```

Reply:
\*fixed a bunch missing spaces/new lines

Reply:
BTW is the username always the email address in OSF @aerubanov?

Reply:
Fixed this.